dist/images/Dockerfile.fedora

```diff
@@ -9,25 +9,93 @@
 # are built locally and included in the image (instead of the rpm)
 #
 
-FROM fedora:41
+ARG OVN_FROM=koji
+
+FROM fedora:41 AS ovnbuilder
 
 USER root
 
 ENV PYTHONDONTWRITEBYTECODE yes
 
+# Install tools that are required for building ovs/ovn.
+RUN INSTALL_PKGS="git rpm-build dnf-plugins-core" && \
+    dnf install --best --refresh -y --setopt=tsflags=nodocs $INSTALL_PKGS
+
+# Clone OVS Source Code.
+ARG OVS_REPO=https://github.com/openvswitch/ovs.git
+ARG OVS_BRANCH=main
```

Comment:
There is no option to customize this or am I missing something? Why not use a stable OVS version?

Reply:
> There is no option to customize this or am I missing something? 
This PR just center on customize OVN, didn't want to make it more complicate by also customize OVS.

> Why not use a stable OVS version?
This is the version used at removed Dockerfile.fedora.dev, but maybe is not the proper one to use here,  @dceara which OVS version should we use with custom OVNs ?

Reply:
That's just to determine what OVS version to use at runtime, that is, for ovsdb and ovs-vswitchd (the OVS version used for libraries is checked out separately from the OVS submodule in the OVN repo).  In practice OVS main should work fine.  However, if you want to use a stable branch instead we do this in core ovn:

https://github.com/ovn-org/ovn/blob/0c120feaea4c3d3299e5ade3155b1003c6a4a7e9/.github/workflows/test.yml#L207-L212

```
        git checkout \
          $(git branch -a -l '*branch-*' | sed 's/remotes\/origin\///' | \
            sort -V | tail -1)
```

We could do the same thing here if you wish.  It would checkout the most recent OVS stable branch tip.

Reply:
Going to use a version of this that do not need to clone the repo 
```
git ls-remote https://github.com/ovn-org/ovn -l '*branch-*' | awk '{print $2}' | sed "s#refs/heads/##g" |sort -V | tail -1
```